src/main/java/seedu/address/model/person/Person.java

```diff
@@ -18,23 +16,23 @@ public class Person {
 
     // Identity fields
     private final Name name;
-    private final Phone phone;
+    private final ClassGroup classGroup;
     private final Email email;
-
-    // Data fields
-    private final Address address;
-    private final Set<Tag> tags = new HashSet<>();
+    private final Phone phone;
```

Comment:
For classGroup and phone, I am wondering if they should also be Optional.
A student might not have found a classGroup at the start of the semester, and some students might not want to share their phone number and would rather communicate through telegram.

Reply:
I think ClassGroup can be compulsory as when the TA adds their students, they know which group they'd be in already I guess.

But for phone definitely I think we can make it optional